### DISCUSSION TITLE
Difference between Faikin and Faikin2 PCB

### DISCUSSION THREAD

**Discussion by ruimmsimoes** [revk/xESP32-Faikout#95] [General]
Hello,

What is the difference between Faikin and Faikin2 PCB?

Are they both in a stable version?
I ask this because I see that you have been updating them in the last weeks.

Have you ordered any batch of Faikin2 from JLCPCB?
I’m considering ordering from JLCPCB because I need 5 devices.

**Comment by revk** [comment]
Faikin2 is later, and will replace Faikin once I get the samples later this week.

**Comment by revk** [comment]
I got the Faikin2 samples and they suffer noise on Rx so do not work without modification. Design updated, and awaiting on new samples.

**Reply by ruimmsimoes** [reply]
Thanks for the feedback.
When you get the new samples let me know if they are ok.

**Comment by lorbetzki** [comment]
intersting, i have a bunch of PCB of Faikin2 with files from July 18 and these running well on my three devices. Which modification do you mean?

**Reply by revk** [reply]
I have been moving the names, i.e. Faikin should be the "current working" and Faikin2 is the "under development", so I will replace Faikin with Faikin2 when tested and working. So not sure of the ones from July 18 - I'd have to check.

This is the problem design - all those passives right by the inductor...

![PICT0021](https://github.com/revk/ESP32-Faikin/assets/996983/dc839e08-ea61-47de-adfb-64256d06dee7)

This is the new layout i am testing this week...

![Faikin](https://github.com/revk/ESP32-Faikin/assets/996983/b2dfe731-d2c5-461b-ba80-57b05fcb7637)

Though I am thinking of being even more cautious with the tracks.

**Comment by lorbetzki** [comment]
Intersting, i am really sure that i downloaded Faikin2 and ordered, but my PCB looks like Faikin 1. 

![20230823_140348](https://github.com/revk/ESP32-Faikin/assets/57233317/e787059d-b162-43a5-9898-4eecf3747fdd)

Maybe it makes sense to rename Faikin2 in Faikin2_Developement?

**Reply by revk** [reply]
Yeh I had Faikin, faikin2 and even faikin3 before but changed my mind about keeping old PCBs (other than in git history). So yea, what you have is fine, just older model.

I should at least add some explanation.

**Comment by revk** [comment]
My samples work - I have updated to be even more cautious though, and expect new samples next week.
